exporter/collector/integrationtest/testdata/fixtures/metrics/google_managed_prometheus.json

```diff
@@ -379,10 +379,88 @@
                            }
                         ]
                      }
+                  },
+                  {
+                     "name":"otlp.test.gauge",
+                     "description":"Normal metric name",
+                     "unit":"seconds",
+                     "gauge":{
+                        "dataPoints":[
+                           {
+                              "timeUnixNano":"1649443516286000000",
+                              "asDouble":0.00699415
+                           }
+                        ]
+                     }
+                  },
+                  {
+                     "name":"workload.googleapis.com/otlp.test.prefix1",
+                     "description":"URI-style metric name",
+                     "unit":"seconds",
+                     "gauge":{
+                        "dataPoints":[
+                           {
+                              "timeUnixNano":"1649443516286000000",
+                              "asDouble":0.00699415
+                           }
+                        ]
+                     }
+                  },
+                  {
+                     "name":".invalid.googleapis.com/otlp.test.prefix2",
+                     "description":"URI-style metric name with invalid domain",
+                     "unit":"seconds",
+                     "gauge":{
+                        "dataPoints":[
+                           {
+                              "timeUnixNano":"1649443516286000000",
+                              "asDouble":0.00699415
+                           }
+                        ]
+                     }
+                  },
+                  {
+                     "name":"otlp.test.prefix3/workload.googleapis.com/abc",
+                     "description":"Broken URI-style metric name",
+                     "unit":"seconds",
+                     "gauge":{
+                        "dataPoints":[
+                           {
+                              "timeUnixNano":"1649443516286000000",
+                              "asDouble":0.00699415
+                           }
+                        ]
+                     }
+                  },
+                  {
+                     "name":"WORKLOAD.GOOGLEAPIS.COM/otlp.test.prefix4",
+                     "description":"Uppercase URI-style metric name",
+                     "unit":"seconds",
+                     "gauge":{
+                        "dataPoints":[
+                           {
+                              "timeUnixNano":"1649443516286000000",
+                              "asDouble":0.00699415
+                           }
+                        ]
+                     }
+                  },
+                  {
+                     "name":"WORKLOAD.googleapis.com/otlp.test.prefix5",
```

Comment:
Can you add a test for `prometheus.googleapis.com/foo` to make it explicit that it will not be stripped ?